crossdock/src/driver.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+/node_modules/.bin/babel-node /crossdock/src/http_server.js &
+/node_modules/.bin/babel-node /crossdock/src/tchannel_server.js &
```

Comment:
you already have index.js below that seems to start them all in one process, so why two different ways?